### PR TITLE
Update Test for `createObject` Function

### PR DESCRIPTION
**Pull request by khadijahazward**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

Related Pull Requests (remove if not relevant)
- Pull request 1
- Pull request 2

One line release note: 
- One line describing the feature/improvement/fix made by this PR 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Ballerina Version:
* Operating System:
* Java SDK: 

# Checklist:

### Security checks
 - [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by tharindu-nw — APPROVED** [review]

File: ballerina/tests/test.bal [hunk 1]
```diff
@@ -50,6 +50,19 @@ function testCreateBucket() {
...
+    dependsOn: [testCreateBucket]
+}
+function testCreateObjectWithMetadata() returns error? {
+    map<string> metadata = {
+        "Description" : "This is a text file",
+        "Language" : "English"
+    };
+    
+    Client amazonS3Client = check new(amazonS3Config);
+    _ = check amazonS3Client->createObject(testBucketName, fileName, content, userMetadataHeaders = metadata);
```
Comment (tharindu-nw) [inline ballerina/tests/test.bal:63]: Shall we test if the metadata was correctly added by getting the object and checking the metadata?

**Review by tharindu-nw — CHANGES_REQUESTED** [review]

File: ballerina/tests/test.bal (hunk 1 above)
Comment (khadijahazward) [reply ballerina/tests/test.bal:63]: The getObject function does not retrieve user-defined metadata. I only updated the createObject function to add user-defined metadata.

Comment (tharindu-nw) [reply ballerina/tests/test.bal:63]: Can we do something similar to what we did for the doc-ui GET endpoint? Calling the s3 url with range=0 and check the headers of the response?

```
http:Response|error response = httpClient->get("objectUrl", {"Range": "bytes=0-0"});
```

Comment (tharindu-nw) [reply ballerina/tests/test.bal:63]: This improvement will be tracked in https://github.com/ballerina-platform/ballerina-library/issues/6989

**Review by tharindu-nw — APPROVED** [review]

**Pull request merged**